<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Generar Valores</string>
       </attribute>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Número de ciudades:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Número de Rutas:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnGenerateTreePrim">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>150</y>
          <width>301</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Generar ciudad con carreteras de doble sentido</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnGenerateTreeD">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>301</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Generar ciudad con carreteras mixtas</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="numberCities">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="numberAristas">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Grafo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="0" rowspan="2" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="lblprim">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Tiempo mínimo para la ruta cobertora de la ciudad:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="frame_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1600000</width>
            <height>1600000</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Agregar carretera</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>100</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QComboBox" name="cbFrom">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbOption">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbTo">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Tiempo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="spbWeight">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="btnAddArista">
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Opciones</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QRadioButton" name="btnhideW">
             <property name="text">
              <string>Ocultar Peso</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Caminos mínimos</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QRadioButton" name="btnShowW">
             <property name="text">
              <string>Mostrar Peso</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btnprim">
             <property name="text">
              <string>Ruta que conecta todas las ciudades</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Insertar ciudades</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Nombre:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Nombre"/>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Costo de hospedaje:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Costo"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Índice de satisfacción (estrellas):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_NumeroEstrellas"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_AgregarCiudad">
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2" rowspan="6">
         <widget class="QFrame" name="frame">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_5">
              <attribute name="title">
               <string>Dijkstra</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Establezca el nodo de origen:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBox_NodoInicial"/>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButton_DeterminarDijkstra">
                 <property name="text">
                  <string>Determinar</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="3">
                <widget class="QPlainTextEdit" name="plainTextEdit_Impresion_Dijkstra"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Información de las ciudades</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTableWidget" name="cityTable"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Matriz de adyacencia</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="0">
                <widget class="QTableWidget" name="adjacencyTable"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Muestra la ruta más corta entre dos ciudades</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Datos de la ciudad seleccionada</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="lblname">
                <property name="text">
                 <string>sdsd</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Número de Estrellas:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Nombre:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="lblcost">
                <property name="text">
                 <string>sdsdsd</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="lblstars">
                <property name="text">
                 <string>sdsdsds</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Costo:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="lblpath">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
